### PR TITLE
UDX-195-troubleshooting-guide

### PR DESCRIPTION
**Pull request by jaffrepaul**
Adds a troubleshooting page to the CT docs to cover commonly faced issues. 

Combines:
- https://cypress-io.atlassian.net/browse/UDX-195
- https://cypress-io.atlassian.net/browse/UDX-193
- https://cypress-io.atlassian.net/browse/UDX-62

### CODE REVIEW

File: content/guides/getting-started/component-test-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: Component Troubleshooting
+---
+
+There are times when you will encounter errors or unexpected behavior with
```
Comment (jaffrepaul) [inline content/guides/getting-started/component-test-troubleshooting.md:5]: Considering adding a section like this at the top as a general introduction. I just pulled this from https://docs.cypress.io/guides/references/troubleshooting but we can pull this into a partial if using the same content makes sense.

**Review by elylucas — CHANGES_REQUESTED** [review]
Got a few suggestions. Also, we might want to update the heading's casing to be more consistent. I opened a discussion in slack but feel free to update the headings if you like.

File: content/guides/getting-started/component-test-troubleshooting.md (hunk 1 above)
Comment (jaffrepaul) [reply content/guides/getting-started/component-test-troubleshooting.md:5]: dig it.

**Review by elylucas — APPROVED** [review]
🤙

**Pull request merged**